docs/multicluster/antctl.md

```diff
@@ -36,57 +36,96 @@ To see the usage examples of these commands, you may also run `antctl mc get [su
 
-To see the usage examples of these commands, you may also run `antctl mc add [subcommand] --help`.
-
-## antctl mc delete
+To see the usage examples of these commands, you may also run `antctl mc deploy [subcommand] --help`.
 
-`antctl mc delete` command can delete resources in an Antrea Multi-cluster ClusterSet.
+## antctl mc init
 
-+ `antctl mc delete clusterclaims` command can delete the two ClusterClaims in a specified Namespace. One for the leader or member cluster, and another for the ClusterSet.
-+ `antctl mc delete clusterset` command can delete a ClusterSet in a leader or member cluster.
-+ `antctl mc delete member-cluster` command can delete a member cluster in a specified Antrea Multi-cluster ClusterSet.
+`antctl mc init` command initializes an Antrea Multi-cluster ClusterSet in a leader cluster. It will create a
+ClusterSet and ClusterClaims for the leader cluster. If the `--output-file` option is specified, the config arguments
+for member clusters to join the ClusterSet will be saved to the specified file.
 
 ```bash
-antctl mc delete clusterclaims [-n NAMESPACE]
-antctl mc delete clusterset [NAME] [-n NAMESPACE]
-antctl mc delete membercluster [MEMBER_CLUSTER_ID] [-n NAMESPACE] [--clusterset CLUSTERSET]
+antctl mc init [-n NAMESPACE] [--clusterset CLUSTERSET_ID] [--clusterid CLUSTERID] [--create-token] [--output-file OUTPUT_FILE]
 ```
 
-To see the usage examples of these commands, you may also run `antctl mc delete [subcommand] --help`.
+To see the usage examples of this command, you may also run `antctl mc init --help`.
 
-## antctl mc deploy
+## antctl mc join
 
-`antctl mc deploy` command can deploy Antrea Multi-cluster Controller to a leader or member cluster.
+`antctl mc join` command lets a member cluster join an existing Antrea Multi-cluster ClusterSet. It will create a
+ClusterSet and ClusterClaims for the member cluster. Users can use command line options or a config file (which can
+be the output file of the `anctl mc init` command) to specify the ClusterSet join arguments.
 
-+ `antctl mc deploy leadercluster` command can deploy Antrea Multi-cluster Controller to a leader cluster, and define all the CRDs the leader cluster needed.
-+ `antctl mc deploy membercluster` command can deploy Antrea Multi-cluster Controller to a member cluster, and define all the CRDs the member cluster needed.
+When the config file is provided, the command line options may be overridden by the file. A token is needed for a
+member cluster to access the leader cluster API server. Users can either specify a pre-created token Secret with the
+`--token-secret-name` option, or pass a Secret manifest to create the Secret with either the `--token-secret-file`
+option or the config file.
 
 ```bash
-antctl mc deploy leadercluster [--antrea-version ANTREA_VERSION] [-n NAMESPACE] [-f PATH_TO_MANIFEST]
-antctl mc deploy membercluster [--antrea-version ANTREA_VERSION] [-n NAMESPACE] [-f PATH_TO_MANIFEST]
+antctl mc join --clusterset=[CLUSTERSET_ID] \
```

Comment:
Should we remove "[]" around the values? "[]" usually means optional.

Reply:
Sure. Removed the "[]".